.babelrc

```diff
@@ -1,4 +1,13 @@
 {
   "presets": ["@babel/preset-env", "@babel/preset-react"],
-  "plugins": ["@loadable/babel-plugin"]
+  "plugins": [
+    "@loadable/babel-plugin",
+    [
+      // https://medium.com/@sherryhsu/how-to-change-relative-paths-to-absolute-paths-for-imports-32ba6cce18a5
+      "module-resolver",
```

Comment:
Our tests do not go through webpack transformation and therefore will not recognize the absolute path aliases if we have configured this in webpack only.

Since Jest tests go through Babel for transpile, we can use a babel module resolver to resolve the relative directives into absolute paths.